src/test/java/org/openmainframeproject/cobolcheck/ExpanderTest.java

```diff
@@ -488,15 +549,138 @@ public void it_inserts_code_correctly_when_each_call_in_series_has_an_exception_
+                    "           MOVE \"para with unmock call\"                                       " + Constants.NEWLINE +
+                    "               TO UT-TEST-CASE-NAME                                            " + Constants.NEWLINE + 
+                    "           PERFORM UT-INITIALIZE-MOCK-COUNT                                    " + Constants.NEWLINE +
+                    "            PERFORM 100-PARA                                                   " + Constants.NEWLINE +
+                    "           MOVE SPACES                                                         " + Constants.NEWLINE +
+                    "               TO UT-TEST-CASE-NAME                                            " + Constants.NEWLINE +
+                    "           PERFORM UT-AFTER-EACH                                               " + Constants.NEWLINE +
+                    "       UT-BEFORE-EACH.                                                          " + Constants.NEWLINE +
+                    "      *****************************************************************         " + Constants.NEWLINE +
+                    "      *This is performed before each Test Case                                  " + Constants.NEWLINE +
+                    "      *****************************************************************         " + Constants.NEWLINE +
+                    "           CONTINUE                                                             " + Constants.NEWLINE +
+                    "           .                                                                    " + Constants.NEWLINE +
+                    "                                                                                " + Constants.NEWLINE +
+                    "       UT-AFTER-EACH.                                                           " + Constants.NEWLINE +
+                    "      *****************************************************************         " + Constants.NEWLINE +
+                    "      *This is performed after each Test Case                                   " + Constants.NEWLINE +
+                    "      *****************************************************************         " + Constants.NEWLINE +
+                    "           CONTINUE                                                             " + Constants.NEWLINE +
+                    "           .                                                                    " + Constants.NEWLINE +
+                    "                                                                                " + Constants.NEWLINE +
+                    "       PROCESS-UNMOCK-CALL.                                                     " + Constants.NEWLINE +
+                    "       .                                                                        " + Constants.NEWLINE +                                                               
```

Comment:
This is a very minor thing, it would be nice to put a CONTINUE inside process-unmock-call.
It will remove a compilation warning on Z.

Reply:
The final period, could also be placed in the wrong area.

Reply:
@Rune-Christensen 
sorry couldn't understand this 
>The final period, could also be placed in the wrong area.